project-root/Assets/Editor/MonsterspawnerEditor.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEditor;
+[CustomEditor(typeof(Monsterspawner))]

```

Comment:
I'm not sure what the goal of this line is? Currently you're trying to extend a namespace that doesn't exist? For future reference, anything related to this game should be under the Hunter namespace, with further derivatives where needed.